lib/season_stats.rb

```diff
@@ -1,5 +1,5 @@
 require_relative 'stats'
-
+require 'pry'
```

Comment:
Looks like this was accidentally left in

Reply:
There is one on runner as well we can get rid of eventually